From ae654c26bea6da7254cc2312fff5330a38e229fa Mon Sep 17 00:00:00 2001
From: Amelie Delaunay <amelie.delaunay@foss.st.com>
Date: Fri, 16 Jul 2021 15:32:16 +0200
Subject: [PATCH 144/529] dt-bindings: phy: stm32: add optional wakeup
 interrupt per port

When a usbphyc phy is used by USBH controller, the wakeup signal is
mapped on EXTI for platform wakeup on STM32MP13.

This patch adds optional interrupt property to usbphyc child nodes.

Change-Id: I9d56fc2521ce25cca9464cda61500cf81cd04fea
Signed-off-by: Amelie Delaunay <amelie.delaunay@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/214513
Reviewed-by: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
Reviewed-by: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 Documentation/devicetree/bindings/phy/phy-stm32-usbphyc.yaml | 4 ++++
 1 file changed, 4 insertions(+)

--- a/Documentation/devicetree/bindings/phy/phy-stm32-usbphyc.yaml
+++ b/Documentation/devicetree/bindings/phy/phy-stm32-usbphyc.yaml
@@ -74,6 +74,10 @@ patternProperties:
       "#phy-cells":
         enum: [ 0x0, 0x1 ]
 
+      interrupts:
+        maxItems: 1
+        description: interrupt used for wakeup when port is used by USBH controller
+
       connector:
         type: object
         $ref: /schemas/connector/usb-connector.yaml
